plugins/shared-react/src/utils/date/format-date.ts

```diff
@@ -0,0 +1,9 @@
+import { format } from 'date-fns';
+
+export function formatDate(date: string | undefined) {
```

Comment:
nit: good to be consistent and use arrow functions and have unit test for it

do we need to have date type as `string | undefined` or have it as just `string`? as the caller should pass a value and for runtime, if it's not there then we should do an early return as you are doing?

Reply:
> nit: good to be consistent and use arrow functions and have unit test for it

https://github.com/janus-idp/backstage-plugins/pull/618#discussion_r1290698458
Ill add a unit test.

> do we need to have date type as string | undefined or have it as just string? as the caller should pass a value and for runtime, if it's not there then we should do an early return as you are doing?

I did it this way for simplicity because one of the values in the nexus plugin is `string | undefined`